components/vision.py

```diff
@@ -95,13 +103,21 @@ def __init__(
         # This has nothing to do with the servo - do it by hand!!
         self.encoder_offset = encoder_offset
 
-        # To find the servo offset, command the servo to neutral in test mode and record the encoder value
-        self.servo_offset = servo_offset
+        # To find the servo offsets, command the servo to neutral in test mode and record the encoder value
+        # Repeat for full range
+        self.servo_offsets = servo_offsets
+
+        relative_rotations = {(r - encoder_offset).radians() for r in rotation_range}
```

Comment:
nit: `rotation_range` is small, so any benefit of this being a set is either negligible or negated (and I don't feel like answering questions about the difference between lists and sets in the middle of build season)

```suggestion
        relative_rotations = [(r - encoder_offset).radians() for r in rotation_range]
```

Reply:
That is just a typo. Was meant to be a list!!